### ISSUE TITLE
Typo reported on Docker Hub - RC_NAME

### ISSUE THREAD

**Issue by colemickens** [ployst/docker#8]
From `thibserot` on Docker Hub:

------------

Hello,
In recreate_pods.sh, looks like there is an omission:
IMAGE=$(kubectl get rc nginx-ssl-proxy-api -o=template --template='{{index .spec.template.spec.containers 0 "image"}}')

Should be:

```
IMAGE=$(kubectl get rc $RC_NAME -o=template --template='{{index .spec.template.spec.containers 0 "image"}}')
```
If you have a github repo where the image is stored, I'll be happy to submit a PR!

**Comment by alexcouper** [comment]
Ah yes, this appears to be hardcoded [here](https://github.com/ployst/docker/blob/master/letsencrypt/recreate_pods.sh#L19).

I'll fix it this week

**Comment by alexcouper** [comment]
Released in [0.0.5](https://hub.docker.com/r/ployst/letsencrypt/tags/)
